tests/publiccloud/registration.pm

```diff
@@ -26,7 +26,7 @@ sub run {
 
     select_host_console();    # select console on the host, not the PC instance
 
-    registercloudguest($args->{my_instance}) if (is_byos());
+    registercloudguest($args->{my_instance}) if (is_byos() || get_var('PUBLIC_CLOUD_FORCE_REGISTRATION'));
```

Comment:
If we're introducing a new variable, please add it also to [variables.md](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md)

Reply:
> If we're introducing a new variable, please add it also to [variables.md](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/variables.md)

Done. thanks for the reminder :).